flows/engine/assets.go

```diff
@@ -3,90 +3,142 @@ package engine
 import (
 	"encoding/json"
 	"fmt"
+	"time"
 
 	"github.com/nyaruka/goflow/flows"
 	"github.com/nyaruka/goflow/flows/definition"
 	"github.com/nyaruka/goflow/utils"
 )
 
-type assets struct {
-	flows    map[flows.FlowUUID]flows.Flow
-	channels map[flows.ChannelUUID]flows.Channel
+type assetContainer struct {
+	assetType  flows.AssetType
+	asset      flows.Asset
+	accessedOn *time.Time
```

Comment:
I'm thinking we'll eventually evict stuff from the asset cache based on when it was last accessed